Update/kage_010.txt

```diff
@@ -1715,20 +1715,20 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 36, "ps3/s08/36/320100434", 256, TRUE);
 	OutputLine(NULL, "「うん。わかってるよ。",
-			NULL, "\"Yeah. I understand.", Line_WaitForInput);
+			NULL, "\"Yeah. I know.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 36, "ps3/s08/36/320100435", 256, TRUE);
 	OutputLine(NULL, "親戚みんなへの見栄で進学校に通わせて、毎日私を勉強漬けにさせたおかーさん」",
-			NULL, " Mom, all you care about is bragging to our relatives about how you're making me go to cram school and study every day.\"", Line_Normal);
+			NULL, " Mom, you're made me go to prep school and study hard every day so you could show off to all our relatives.\"", Line_Normal);
 	ClearMessage();
 
 //夏美rvS08/36/320100436.「出世に響くから、私の入院の理由を貧血症にしてごまかしたおとーさん。kvS08/36/320100437.…私、何か間違ってる？」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#94a567>夏美</color>", NULL, "<color=#94a567>Natsumi</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 36, "ps3/s08/36/320100436", 256, TRUE);
 	OutputLine(NULL, "「出世に響くから、私の入院の理由を貧血症にしてごまかしたおとーさん。",
-			NULL, "\"And Dad, you always worried that the reason for my hospitalization would undercut your career advancement.", Line_WaitForInput);
+			NULL, "\"And Dad, you lied that I was hospitalized for anemia because you were worried the real reason would undercut your career advancement.", Line_WaitForInput);
```

Comment:
```suggestion
			NULL, "\"And Dad, you lied and said I was hospitalized for anemia because you were worried the real reason would undercut your career advancement.", Line_WaitForInput);
```